vercel.json

```diff
@@ -224,6 +224,18 @@
             "source": "/d/relation-mode",
             "destination": "https://www.prisma.io/docs/concepts/components/prisma-schema/relations/relation-mode"
         },
+        {
+            "source": "/d/postgres-set-null",
+            "destination": "https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#types-of-referential-actions"
+        },
+        {
+            "source": "/d/mysql-set-default",
+            "destination": "https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#types-of-referential-actions"
+        },
```

Comment:
```suggestion
        {
            "source": "/d/postgres-set-null",
            "destination": "https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#special-cases-for-referential-actions"
        },
        {
            "source": "/d/mysql-set-default",
            "destination": "https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#special-cases-for-referential-actions"
        },
```

Reply:
This mirrors the preview link mentioned in: https://github.com/prisma/docs/pull/4132#issuecomment-1327776745